### ISSUE TITLE
Fully resumable checkpoint

### ISSUE THREAD

**Issue by Engeryu** [Engeryu/EdgeVision-HW-Bridge#26]
Checkpoint now stores model, optimizer, scheduler state_dict, epoch and best_acc.\nCommit: feat(ml): AMP mixed precision, torch.compile, global seed, resumable checkpoint
